### PR TITLE
SYS-154 improved alias not found message

### PR DESCRIPTION
**Pull request by willyko**

### CODE REVIEW

File: src/alias.cpp [hunk 1]
```diff
@@ -2445,11 +2445,11 @@ UniValue aliasinfo(const JSONRPCRequest& request) {
 	vector<unsigned char> vchAlias = vchFromValue(params[0]);
 	ToLowerCase(vchAlias);
 	CAliasIndex txPos;
-	if (!paliasdb || !paliasdb->ReadAlias(vchAlias, txPos))
+	if (!paliasdb)
```
Comment (jnessbcf) [inline src/alias.cpp:2448]: Should be a unit test for both conditions, one for each?

Comment (willyko) [reply src/alias.cpp:2448]: i don't think the unit test checks for the correct error message

Comment (jnessbcf) [reply src/alias.cpp:2448]: Right, I was suggesting adding two unit tests, one for each of the two conditions.  The existing "unit test" is more like an integration test, from what I recall.

**Comment by sidhujag** [comment]
ACK

**Pull request merged**